sdk/eventhub/Microsoft.Azure.EventHubs/tests/Infrastructure/EventHubScope.cs

```diff
@@ -0,0 +1,300 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Runtime.CompilerServices;
+using System.Security.Authentication;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.EventHub;
+using Microsoft.Azure.Management.EventHub.Models;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.Rest;
+using Polly;
+
+
+namespace Microsoft.Azure.EventHubs.Tests
+{
+    // Ported from Track Two EventHubScope.cs
```

Comment:
This really doesn't belong hanging on top of the document comments; please remove.

Reply:
Removed all comments to fit track one styles, and make this internal like TestUtility.